### PR TITLE
tools: add a simple DM usage scenario

### PR DESCRIPTION
**Pull request by lilin90**
@csuzhangxc @CaitinChen PTAL
Via: https://github.com/pingcap/tidb-tools/pull/151

### CODE REVIEW

File: tools/dm-simple-synchronization-scenario.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+| Schema | Tables|
+|:------|:------|
+| user_north | information, log|
+| user_east  | information, log|
+| user_south | information, log|
+| store | store_bj, store_tj, store_sh, store_suzhou, store_gz, store_shenzhen |
+
+## Synchronization solution
+
+- To satisfy the i, ii, and iii items of the first synchronization requirement, configure the table routing rule as follows:
```
Comment (csuzhangxc) [inline tools/dm-simple-synchronization-scenario.md]: how about? and more below.
```suggestion
- To satisfy the i, ii, and iii items of the first synchronization requirement, configure the table routing rules as follows:
```

and any link to `table routing rule`?

File: tools/dm-simple-synchronization-scenario.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    ```yaml
+    filters:
+    ...
+    store-filter-rule:
+        schema-pattern: "store"
+        events: ["drop database", "truncate table", "drop table", "delete"]
+        action: Ignore
+    ```
+
+    > **Note:** `store-filter-rule` is different from `log-filter-rule & user-filter-rule`. `store-filter-rule` is the schema of the whole `store`, while `log-filter-rule & user-filter-rule` is the schema of the `user`.`log` tables.
```
Comment (csuzhangxc) [inline tools/dm-simple-synchronization-scenario.md]: how about
```
`store-filter-rule` is a rule for the whole `store` schema, while `log-filter-rule` and `user-filter-rule` are rules for the `log` table in `user` schema.
```

**Comment by lilin90** [comment]
@csuzhangxc @CaitinChen Thanks! PTAL again

**Comment by csuzhangxc** [comment]
LGTM

**Pull request merged**